sites/partners/__tests__/PaperListingForm/sections/DetailUnits.test.tsx

```diff
@@ -28,6 +28,7 @@ describe("DetailUnits", () => {
     expect(results.getByText("None")).toBeInTheDocument()
   })
 
+  // skipping for now until we can solve the getByRole issue
```

Comment:
What issue is this?

Reply:
Nice catch, that was a note to myself. I was able to resolve the issue so removed the skip but forgot to remove the comment